### DISCUSSION TITLE
DaemonSet engine-image doesn't spread across new nodes

### DISCUSSION THREAD

**Discussion by Tarasovych** [longhorn/longhorn#5700] [Q&A]
I use Longhorn 1.4.1 in EKS 1.24 cluster. I have managed node group. For some reason engine-image DaemonSet doesn't spread across new nodes in the node group, when I scale the node group up (manually, for example).

Any suggestions where should I start debugging?

Manager logs only tell me, that
```
"CheckEngineImageReadiness: nodes [ip-xx-xx-xx-xx.xx-xxxx-x.compute.internal] don't have the engine image longhornio/longhorn-engine:v1.4.1"
```
and there are no pending pods in `longorn-system` namespace. I also tried `rollout restart` without no success

**Comment by mantissahz** [comment]
Could you check about that the new node has been any tainted?
